### PR TITLE
develop/print-node_info

### PR DESCRIPTION
**Pull request by WinnieXU1**
1.add the code in cli.py: send cmd "infrasim node info" will print node_info;
2.add a "start_info_destroy_info" test.

### CODE REVIEW

File: infrasim/cli.py [hunk 1]
```diff
@@ -162,6 +164,50 @@ def destroy(self, node_name):
...
+            node_info_net = node_info['compute']['networks']
+            node_info_stor = node_info['compute']['storage_backend']
+
+            print "node name:          {}\n" \
+                  "type:               {}\n" \
+                  "memory size:        {}\n" \
+                  "sol_enable:         {}\n" \
+                  "cpu quantities:     {}\n" \
+                  "cpu type:           {}" . \
+                format(node_name, node_info['type'], node_info['compute']['memory']['size'], node_info['sol_enable'],
```
Comment (XiaowenJiang) [inline infrasim/cli.py]: recommend to use "{:<15}".format(a) format to fill the white spaces.

Comment (WinnieXU1) [inline infrasim/cli.py]: have a try right now

**Comment by fub2** [comment]
Overall looks good. 
@WinnieXU1 can you paste example responses in this PR? It would be even great if you can paste response when network configuration are of different types, say:
1. NAT
2. bridge - like what our standard OVA does

**Comment by WinnieXU1** [comment]
`node name:          default
type:               quanta_d51
memory size:        1024
sol_enable:         1
cpu quantities:     2
cpu type:           Haswell
network(s):         ----------------------
                    device   mode    name  
                    e1000    nat    ens160 
storage backend:    ------------------------------
                    type   max drive   drive size 
                    ahci   8           8 `

**Comment by WinnieXU1** [comment]
'''
node name:          default
type:               quanta_d51
memory size:        1024
sol_enable:         1
cpu quantities:     2
cpu type:           Haswell
network(s):         ----------------------
                    device   mode    name  
                    e1000    nat    ens160 
storage backend:    ------------------------------
                    type   max drive   drive size 
                    ahci   8           8 
'''

**Comment by WinnieXU1** [comment]
node_info-response-bridge
node_info-response-nat

**Comment by WinnieXU1** [comment]
node_info-response-bridge
node_info-response-nat

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_start_info_destroy_info 
ErrorType: exceptions.AssertionError 
ErrorMessage:  
```

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_start_info_destroy_info 
ErrorType: exceptions.AssertionError 
ErrorMessage:  
```

**Comment by turtle-fly** [comment]
retest it please

**Comment by turtle-fly** [comment]
retest this please

**Pull request merged**